addons/common/fnc_getCfgDataRandom.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getCfgDataRandom
+
+Description:
+    This function extracts data from a config property.
+    If it is an Array, it will select a random entry from the array, otherwise it will simply return the provided data.
```

Comment:
```suggestion
    If it is an Array, it will select a random entry from the array (nil if empty), otherwise it will simply return the provided data.
```